frontend/app/utils/env.server.ts

```diff
@@ -12,11 +12,16 @@ function tryOrElseFalse(fn: () => unknown) {
   catch { return false; }
 }
 
+// @see app/mocks/node.ts
+const mockNames = ['cct', 'lookup', 'power-platform', 'raoidc', 'wsaddress'];
+
 // refiners
+const isValidMockName = (val: Array<string>) => val.every((val) => mockNames.includes(val));
 const isValidPublicKey = (val: string) => tryOrElseFalse(() => publicKeyPemToCryptoKey(val));
 const isValidPrivateKey = (val: string) => tryOrElseFalse(() => privateKeyPemToCryptoKey(val));
 
 // transformers
+const toArray = (val?: string) => val?.split(',').map((val) => val.trim()) ?? [];
```

Comment:
```suggestion
const csvToArray = (val?: string) => val?.split(',').map((val) => val.trim()) ?? [];
```

Reply:
I took your advice and renamed some of the variables to be more descriptive. Please re-review at your leisure.